core-services/prow/02_config/_plugins.yaml

```diff
@@ -477,6 +477,10 @@ approve:
   - openshift-priv/cloud-provider-azure
   - openshift/cloud-provider-azure
   require_self_approval: false
+- commandHelpLink: ""
```

Comment:
likely need an entry like the following in this file:
```
     ci-operator/config/**/cluster-logging-operator/openshift*-cluster-logging-operator-feature*.yaml:
       clusters:
         app.ci:
         - ci
       gzip: true
       name: ci-operator-misc-configs
```

Reply:
done